3648/1/gem_stat.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'nokogiri'
+require 'open-uri'
+require 'httparty'
+
+AUTH_TOKEN = '43a345ee370fdfafc836b3534b4311a3a8ee45df'
+
+class GemStatictic
+  attr_reader :gem_name
+  def initialize(gem_name)
+    @gem_name = gem_name
+  end
+
+  def create_stat
+    {
+      gem_name: gem_name,
+      used_by: find_used_by,
+      watch_by: find_wath_by,
```

Comment:
Spelling mistake. And if you decided to use past tense, use it everywhere. By past tense I mean `watched_by`